docs/supported_platforms_and_configurations.md

```diff
@@ -64,7 +64,7 @@ For more information see [conan-io/conan-docker-tools](https://github.com/conan-
 
 ## MacOS
 
-- Compilers: Apple-clang versions 10.0.1, 11.0.0, 12.0.0, 12.0.0 (three latest versions, we will rotate the older when a new compiler version is released)
+- Compilers: Apple-clang versions 11.0.0, 12.0.0, 12.0.0 (three latest versions, we will rotate the older when a new compiler version is released)
```

Comment:
```suggestion
- Compilers: Apple-clang versions 11.0.0, 12.0.0, 12.0.0 (three latest versions, we will rotate the older when a new compiler version is released)
```

12.0.0, 12.0.0?

Reply:
If only 11 and 12 are supported, then the text should be updated as well.

Reply:
the text is auto-generated, it pointless to edit it manually, since bot will overwrite it and create new PR again
all the inconsistences has to be addressed in bot's generator